web/src/components/ImageSurveyCell/ImageSurveyCell.tsx

```diff
@@ -134,14 +137,29 @@ const ImageSurveyComponent = ({
 
   const onSubmit: SubmitHandler<PlainImageSurveyValues> = async (data) => {
     const privateRank = parseInt(data[IS_PRIVATE_QUESTION_GROUP_A])
+
+    // The data returns an object with tags and input, we need a single string with the tags.
+    // https://stackoverflow.com/questions/52936112/react-js-need-to-split-the-state-object-being-retrieved
+    const publicElement = Object.values(
+      data[PUBLIC_ELEMENTS_QUESTION_GROUP_A]
+    )[0].toString()
```

Comment:
Yes this is normal, you told TS on line 91-92 that these properties would be strings. but their type changed. You should update the type on line 91-92 and it will work.